packages/database/lib/modular/query.js

```diff
@@ -239,6 +239,10 @@ export function get(query) {
   return query.once('value');
 }
 
+export function off(query, eventType, callback) {
+  throw new Error("off() is not implemented");
```

Comment:
similar to enableLogging - if it is not implemented it seems that should be noted in the docs for the method in query.d.ts? And also same as enableLogging, any reason not to plumb it through on web/other implementation (if it is not already)